release notes/v0.47.0.md

```diff
@@ -37,6 +37,10 @@ Thanks, @sapphire-janrain, for contribution!
 
 In some workflows the reflection call should also include some metadata. This PR adds [a new connection parameter `reflectMetadata`](https://k6.io/docs/javascript-api/k6-net-grpc/client/client-connect/#connectparams) that allows to specify the metadata to be sent with the reflection call.
 
+### Add support for browser module's `page.on('console')` [xk6-browser#1006](https://github.com/grafana/xk6-browser/pull/1006)
```

Comment:
I'm wondering if this could be shortened by dropping "browser module's", since the link to the PR contains `xk6-browser`. Maybe something like:

```suggestion
### Add `page.on('console')` [xk6-browser#1006](https://github.com/grafana/xk6-browser/pull/1006)
```

WDYT?

Reply:
I thought it made sense to explicitly mention the "browser module" so it's better scoped in the context of the whole k6 release notes. It's true that there is a reference to `xk6-browser` repo in the PR link, but I'm not sure if that is enough for good "readability".
I'm OK shortening it to fit your suggestion, but mentioning that this was the reasoning behind it. I would like to hear other opinions on making this explicit distinctions in the release notes points that relate to the browser module, also for future release notes.

Reply:
Happy to wait for some clarification on this 👍 

Just wanted to point out that in previous release notes we didn't add "browser module".

Also, we used `browser` instead of `xk6-browser` in the PR link.

Reply:
> Also, we used browser instead of xk6-browser in the PR link in our previous release notes.

Right! Let me change that.

Reply:
It feels better to me if we keep the browser in the title as it makes it more explicit 👍  For example, [this previous release notes](https://github.com/grafana/k6/blob/fb71278545d5626b9d3fd7a3f7b56b82b528bdcc/release%20notes/v0.46.0.md?plain=1#L84) include `gRPC` despite the repo name is already on the link.

Reply:
Happy to leave it in 👍  Last suggestion from me on this 🙂 

```suggestion
### Add `page.on('console')` to the browser module [xk6-browser#1006](https://github.com/grafana/xk6-browser/pull/1006)
```